### ISSUE TITLE
Half-Screen Home Page Icons

### ISSUE THREAD

**Issue by KCoterwas** [Capping-CPCA/Web-App#24]
As a half-screen, the icons are in a way where you have to scroll to see them all. Should that be a thing or can we display them all at once?

**Comment by algozine** [comment]
I don't think we should worry about mobile devices .. and doing this level of UI updates is a nice to have ... low priority
